uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -0,0 +1,492 @@
+define([
+  'leave-absences/shared/modules/controllers',
+  'common/lodash',
+  'common/moment',
+  'common/services/api/option-group',
+  'common/services/hr-settings',
+], function (components, _, moment) {
+  'use strict';
+
+  components.controller('LeaveRequestPopupCtrl', [
+    '$log', '$q', '$uibModalInstance', 'AbsencePeriod', 'AbsenceType', 'Entitlement',
+    'Calendar', 'LeaveRequestInstance', 'LeaveRequest', 'api.optionGroup', 'baseData',
+    'PublicHoliday', 'HR_settings', '$rootScope',
```

Comment:
As before, please let `$rootScope` in alphabetical order alongside the other angular-native modules